0000-template.md

```diff
@@ -1,102 +1,40 @@
-> Here are some tools that we often use:
->
-> - [Graphviz](http://graphviz.it/#/gallery/structs.gv)
-> - [PlantText](https://www.planttext.com)
-
-# Drawbacks
-
-> Why should we _not_ do this? Please consider:
->
-> - implementation cost, both in term of code size and complexity
-> - whether the proposed feature can be implemented in user space
-> - the impact on teaching people how to use CDK
-> - integration of this feature with other existing and planned features
-> - cost of migrating existing CDK applications (is it a breaking change?)
->
-> There are tradeoffs to choosing any path. Attempt to identify them here.
-
-# Rationale and Alternatives
-
-> - Why is this design the best in the space of possible designs?
-> - What other designs have been considered and what is the rationale for not
->   choosing them?
-> - What is the impact of not doing this?
-
-# Adoption Strategy
-
-> If we implement this proposal, how will existing CDK developers adopt it? Is
-> this a breaking change? How can we assist in adoption?
-
-# Unresolved questions
-
-> - What parts of the design do you expect to resolve through the RFC process
->   before this gets merged?
-> - What parts of the design do you expect to resolve through the implementation
->   of this feature before stabilization?
-> - What related issues do you consider out of scope for this RFC that could be
->   addressed in the future independently of the solution that comes out of this
->   RFC?
-
-# Future Possibilities
-
-> Think about what the natural extension and evolution of your proposal would be
-> and how it would affect CDK as whole. Try to use this section as a tool to more
-> fully consider all possible interactions with the project and ecosystem in your
-> proposal. Also consider how this fits into the roadmap for the project.
->
-> This is a good place to "dump ideas", if they are out of scope for the RFC you
-> are writing but are otherwise related.
->
-> If you have tried and cannot think of any future possibilities, you may simply
-> state that you cannot think of anything.
-
-# Implementation Plan
-
-INTENTIONALLY LEFT BLANK: an implementation plan will be added when the RFC is
-scheduled for implementation.
-
-> The implementation plan should analyze all the tasks needed in order to
-> implement this RFC, and the planned order of implementation based on
-> dependencies and analysis of the critical path.
->
-> Either add the plan here or add link which references a separate document
-> and/or a GitHub Project Board (and reference it here) which manages the
-> execution of your plan.
+> One sentence: brief description of the feature in a user-facing terminology.
+
+## Working Backwards
+
+### README
+
+> If this is a new feature, work backwards from the user-facing README artifact.
+
+### RELEASE NOTES
+
+> If this change includes a major breaking change, add a "RELEASE NOTES" section with migration instructions
+
+### PRESS RELEASE
```

Comment:
Not sure this will be clear to external contributors. We should add an example, or some instructions. But honestly i think we can even start without this. The release notes and readme should cover our working backwards strategy.

Reply:
Added details